Library/AppearanceSelectionTableCell.m

```diff
@@ -140,4 +140,10 @@ - (void)updateForType:(int)type {
     }
 }
 
-@end
\ No newline at end of file
+- (void)layoutSubviews {
+    [super layoutSubviews];
+
+    if (self.containerStackView.spacing != (self.bounds.size.width - (60 * [self.options count])) / ([self.options count] + 1)) self.containerStackView.spacing = (self.bounds.size.width - (60 * [self.options count])) / ([self.options count] + 1);
```

Comment:
Not a fan of using ``layoutSubviews`` for stuff like this. Is there a better solution that you could do?